### PR TITLE
Merge operator work

### PR DESCRIPTION
**Pull request by drf**
This merge request brings in all the Operator-related work. Includes a self-contained Ansible Playbook, and an Ansible Operator capable of deploying both Astarte itself, and a Voyager ingress for its components.

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  template:
+    metadata:
+      labels:
+        name: astarte-operator
+    spec:
+      serviceAccountName: astarte-operator
+      containers:
+        - name: astarte-operator
+          # Replace this with the built image name
+          image: astarte/astarte-kubernetes-operator:0.10-snapshot
```
Comment (bettio) [inline deploy/operator.yaml]: is this supposed to be hardcoded?
If yes, we should make sure to have some README that helps people in finding hardcoded values that should be changed.

File: watches.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+- version: v1alpha1
```
Comment (bettio) [inline watches.yaml]: we are not using alpha version.

Comment (drf) [reply watches.yaml]: Alpha is referred to Astarte Kubernetes' CRD revision, so it's correct, and it's Kubernetes standard way of doing things

File: deploy/operator.yaml (hunk 1 above)
Comment (drf) [reply deploy/operator.yaml]: It is. This is the deployment of the operator itself. Arguably, we want to have snapshot in master and 0.10-snapshot in release-0.10

Comment (bettio) [reply deploy/operator.yaml]: We must have some sort of checklist with all the required changes that we should do before releasing stuff. Furthermore I think we should change this every time we are going to tag a release, so some sort of README would be nice.

File: watches.yaml (hunk 2 above)
Comment (bettio) [reply watches.yaml]: So v0.10.0-beta.3 is v1alpha1? So how is going to be called v0.10.0?

**Pull request merged**